ui/src/app/components/ct-available-banner-maybe.tsx

```diff
@@ -53,10 +55,11 @@ export const CTAvailableBannerMaybe = () => {
   const [showBanner, setShowBanner] = useState(false);
   const { profile } = useStore(profileStore);
   const { tiers } = useStore(cdrVersionStore);
+  const { pathname } = useLocation();
```

Comment:
Tested locally - behavior is equivalent plus an improvement: the banner now disappears as soon as navigation to DAR happens, instead of after a short delay (for rendering?)